app/models/rent.rb

```diff
@@ -0,0 +1,4 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
```

Comment:
Good to have the `belongs_to` but do not forget to add the equivalent in `Book` model.